sybil/evaluators/skip.py

```diff
@@ -1,28 +1,28 @@
 from dataclasses import dataclass
-from typing import Optional, Dict
+from typing import Any, Optional, Dict
 from unittest import SkipTest
 
 from sybil import Example, Document
 from sybil.example import NotEvaluated
-from sybil.typing import Evaluator
 
 
 class If:
 
-    def __init__(self, default_reason) -> None:
+    def __init__(self, default_reason: str) -> None:
         self.default_reason = default_reason
 
-    def __call__(self, condition, reason=None):
+    def __call__(self, condition: Any, reason: Optional[str] = None) -> Optional[str]:
         if condition:
             return reason or self.default_reason
+        return None
```

Comment:
With the `mypy` strict mode, all paths need an explicit return statement when the return type is not `None`. If you don't like this, we can ignore the `return` error type in `mypy.ini`. That will not just apply to `return None` statements.